automod/capture/fetch.go

```diff
@@ -36,8 +37,12 @@ func FetchAndProcessRecord(ctx context.Context, eng *automod.Engine, aturi synta
 		return fmt.Errorf("expected a CID in getRecord response")
 	}
 	recCID := syntax.CID(*out.Cid)
+	var ltd lexutil.LexiconTypeDecoder
```

Comment:
I think it would be cleaner to use util.JsonDecodeValue()
maybe make a utility function that returns `(cbg.CBORMarshaler, error)` if we want better than `(any, error)` ?